src/node.js

```diff
@@ -388,6 +396,53 @@
       nextTickQueue.push(obj);
       tickInfo[kLength]++;
     }
+
+    // Internal callbacks that must be run when there is no more micro-tasks
+    // scheduled for this turn.
+    function lateTick(callback) {
+      lateTickQueue.push(callback);
+      scheduleMicrotasks();
```

Comment:
When is `_tickCallback` called? If you do `_lateTick` and don't schedule any microtasks I don't see what'd call `scheduleMicrotasks` for you